### PR TITLE
add ansible lint test

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
:green_apple:

File: playbooks/setup.yaml [hunk 1]
```diff
@@ -3,45 +3,45 @@
...
-    ansible.posix.firewalld:
-      port: "{{ item }}"
-      permanent: true
-      state: enabled
-    loop:
-    - 23443/tcp
-    when:
-    - services['firewalld.service'] is defined
-    - services['firewalld.service']['state'] == 'running'
+    - name: Open Ports in firewalld
```
Comment (evgeni) [inline playbooks/setup.yaml:38]: @ehelms is this even needed? candlepin doesn't need to be open to the world.

Comment (ehelms) [reply playbooks/setup.yaml:38]: If tests pass.. then nope!

**Review by ehelms — APPROVED** [review]

File: playbooks/setup.yaml (hunk 1 above)
Comment (evgeni) [reply playbooks/setup.yaml:38]: https://github.com/theforeman/foreman-quadlet/pull/40

**Pull request merged**